### PR TITLE
Merge wied03/savant_2.0.0

### PR DESCRIPTION
**Pull request by wied03**
PR details

### CODE REVIEW

File: build.savant [hunk 1]
```diff
@@ -40,11 +40,11 @@ project(group: "org.savantbuild", name: "savant-version", version: "2.0.0-RC.7",
...
-dependency = loadPlugin(id: "org.savantbuild.plugin:dependency:2.0.0-RC.6")
-java = loadPlugin(id: "org.savantbuild.plugin:java:2.0.0-RC.6")
-javaTestNG = loadPlugin(id: "org.savantbuild.plugin:java-testng:2.0.0-RC.6")
-idea = loadPlugin(id: "org.savantbuild.plugin:idea:2.0.0-RC.6")
-release = loadPlugin(id: "org.savantbuild.plugin:release-git:2.0.0-RC.6")
+def dependency = loadPlugin(id: "org.savantbuild.plugin:dependency:2.0.0-RC.7")
+def java = loadPlugin(id: "org.savantbuild.plugin:java:2.0.0-RC.6")
+def javaTestNG = loadPlugin(id: "org.savantbuild.plugin:java-testng:2.0.0-RC.6")
+def idea = loadPlugin(id: "org.savantbuild.plugin:idea:2.0.0-RC.7")
+def release = loadPlugin(id: "org.savantbuild.plugin:release-git:2.0.0-RC.6")
```
Comment (robotdan) [inline build.savant]: Not sure I understand the need for this. While this is technically "groovy" it is really supposed to be a Savant build file which doesn't necessarily have to follow general Groovy coding guidelines.

File: savant-version.iml [hunk 2]
```diff
@@ -10,26 +10,26 @@
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="library" name="Groovy 2.4" level="application" />
+    <orderEntry type="library" name="groovy-4.0.22" level="application" />
```
Comment (robotdan) [inline savant-version.iml]: `Groovy X.X` is sort of the standard we have all across our dev environment. We may want to keep this unless there is a technical benefit to changing it.

File: build.savant (hunk 1 above)
Comment (wied03) [reply build.savant]: Yes, fair point, discussed it and reverted.

File: savant-version.iml (hunk 2 above)
Comment (wied03) [reply savant-version.iml]: Agreed. Fixed.

File: savant-version.iml [hunk 3]
```diff
@@ -10,26 +10,26 @@
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="library" name="Groovy 2.4" level="application" />
+    <orderEntry type="library" name="groovy-4.0" level="application" />
```
Comment (robotdan) [inline savant-version.iml]: I think we discussed, perhaps this one got missed?

Comment (wied03) [reply savant-version.iml]: Sorry. missed. wrong convention. Fixed.

**Review by robotdan — APPROVED** [review]

**Pull request merged**